lib/events/athena/querier_test.go

```diff
@@ -45,18 +45,19 @@ func Test_querier_prepareQuery(t *testing.T) {
 		whereTimeRange   = ` WHERE event_date BETWEEN date(?) AND date(?) AND event_time BETWEEN ? and ?`
 	)
 	fromTimeUTC := time.Date(2023, 2, 1, 0, 0, 0, 0, time.UTC)
-	toTimeUTC := time.Date(2023, 3, 1, 0, 0, 0, 0, time.UTC)
+	toTimeUTC := time.Date(2023, 6, 1, 0, 0, 0, 0, time.UTC)
 	fromDateParam := "'2023-02-01'"
 	fromTimestampParam := "timestamp '2023-02-01 00:00:00'"
-	toDateParam := "'2023-03-01'"
-	toTimestampParam := "timestamp '2023-03-01 00:00:00'"
+	toDateParam := "'2023-06-01'"
+	toTimestampParam := "timestamp '2023-06-01 00:00:00'"
```

Comment:
modified only because dynamo pagination test cases are with following timestamp 2023-04-27, so I wanted to include it in range.